src/rendervulkan.cpp

```diff
@@ -147,8 +147,7 @@ struct {
 	bool bNeedsSwizzle;
 	bool bHasAlpha;
 } s_DRMVKFormatTable[] = {
-	{ DRM_FORMAT_XRGB8888, VK_FORMAT_A8B8G8R8_UNORM_PACK32, true, false },
-	{ DRM_FORMAT_ARGB8888, VK_FORMAT_A8B8G8R8_UNORM_PACK32, true, true },
+	{ DRM_FORMAT_XRGB8888, VK_FORMAT_R8G8B8A8_UNORM, true, true },
```

Comment:
`bHasAlpha` should probably be set to false here?

Reply:
typop

Reply:
Fixed.